lib_ecstasy/src/main/x/ecstasy/collections/List.x

```diff
@@ -1468,4 +1468,4 @@ interface List<Element>
 
         return True, maxValue, maxCount;
     }
-}
```

Comment:
Revert unnecessary whitespace change to master files to avoid creating resolutions (even trivial ones)